### PR TITLE
fix(raw-database-access): unsafe & safe updates

### LINKED ISSUE #2288: Raw queries page needs some work
The raw queries page needs some shuffling about and examples improved upon. Including looking at potential SQL Server examples

### PR DESCRIPTION
**Pull request by millsp**
cc @molebox

### CODE REVIEW

File: content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx [hunk 1]
```diff
@@ -35,7 +35,7 @@ Use cases for raw SQL include:
 const result = await prisma.$queryRaw`SELECT * FROM User`
 ```
 
-The method is implemented as a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates), which makes it easier to use [variables](#using-variables) by providing **placeholders**. The following example includes an `${email}` placeholder:
+The method is implemented as a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates), which allows you to pass a template litteral where you can easily insert your [variables](#using-variables). In turn, we create prepared statements that are safe from SQL injections:
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx]:
```suggestion
The method is implemented as a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates), which allows you to pass a template literal where you can easily insert your [variables](#using-variables). In turn, Prisma creates prepared statements that are safe from SQL injections:
```

File: content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx [hunk 2]
```diff
@@ -197,22 +191,23 @@ When you use `ILIKE`, the `%` wildcard character(s) should be included in the va
...
 
 `$executeRaw` returns the _number of rows affected by a database operation_, such as `UPDATE` or `DELETE`. This function does **not** return database records. The following query updates records in the database and returns a count of the number of records that were updated:
 
 ```ts
-const result: number = await prisma.$executeRaw'UPDATE User SET active = true WHERE emailValidated = true'
+const result: number = await prisma.$executeRaw`UPDATE User SET active = true WHERE emailValidated = true`
 ```
 
-The method is implemented as a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates), which makes it easier to use [variables](#using-variables) by providing placeholders. The following example includes `${emailValidated}` and `${active}` placeholders:
+The method is implemented as a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates), which allows you to pass a template litteral where you can easily insert your [variables](#using-variables). In turn, we create prepared statements that are safe from SQL injections:
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx]:
```suggestion
The method is implemented as a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates), which allows you to pass a template literal where you can easily insert your [variables](#using-variables). In turn, Prisma creates prepared statements that are safe from SQL injections:
```
This seems to be repeated from line 38 which is the exact same string.

**Review by janpio — APPROVED** [review]
Seems to fix a few things that were wrong or confusing. 
Technical details someone else needs to double check though please.

**Comment by molebox** [comment]
> Seems to fix a few things that were wrong or confusing.
> Technical details someone else needs to double check though please.

Isnt @millsp the technical person that we would expect to ask to check these? 😅

**Comment by millsp** [comment]
I think that @Jolg42 and @matthewmueller are the perfect fit for double checking this :pray:

**Comment by janpio** [comment]
Well, we usually want someone to _write_ changes and then someone else to _check_ them - both being Pierre would defeat that purpose 😆

**Comment by molebox** [comment]
> Well, we usually want someone to _write_ changes and then someone else to _check_ them - both being Pierre would defeat that purpose 😆

I should have added this GIF to my comment 😄

ba-dum-tis

**Comment by janpio** [comment]
:swoosh:

**Review by Jolg42 — APPROVED** [review]

**Pull request merged**